docs/core_docs/docs/integrations/platforms/google.mdx

```diff
@@ -69,9 +74,80 @@ const input2 = [
 const res = await visionModel.invoke(input2);
 ```
 
-The value of image_url must be a base64 encoded image (e.g., data:image/png;base64,abcd124).
+:::tip
+Click [here](/docs/integrations/chat/google_generativeai) for the `@langchain/google-genai` specific integration docs
+:::
 
-### Vertex AI
+</TabItem>
+
+<TabItem value="vertexai" label="VertexAI" default>
+<IntegrationInstallTooltip></IntegrationInstallTooltip>
+
+```bash npm2yarn
+npm install @langchain/google-vertexai
+```
+
+Configure your API key.
```

Comment:
Vertex AI doesn't use an API key. The Python docs skip over auth completely (https://python.langchain.com/docs/integrations/platforms/google#vertex-ai)

Generating the service account key is a 15 step process.
Using that key depends on using the gauth vs webauth packages.

Reply:
The documentation for service accounts under chat/google_vertex_ai.mdx are probably the ones that should be used here (minus the API Key parts).